### DISCUSSION TITLE
公式エディタを使用しているとメモリ使用量がどんどん増加してクラッシュする

### DISCUSSION THREAD

**Discussion by matsumoto616** [increments/qiita-discussions#572] [要望・アイデア・不具合]
【環境】
OS: Windows11
PC: Surface pro 9, メモリ16GB
ブラウザ：Edge

【問題】
公式エディタを用いてブラウザ上で記事を編集していると、だんだんとメモリ使用量が増加していきクラッシュします。タスクマネージャーで見たところ、文字を打たずとも画面をスクロールしているだけでメモリ使用量がどんどん増えていきます。なにかメモリリークが起きているように見えるのですが、同様の問題が発生していたりしないでしょうか？（私の環境固有の問題でしょうか？）
![image](https://github.com/increments/qiita-discussions/assets/65050759/9a10a181-0425-419a-bc52-daa58736b3f8)

書いている記事はこちらになります。数式はやや多いですが、画像を貼っているわけでもないので、メモリ使用量がいっぱいになるのは変な気がしています。
https://qiita.com/matsumoto616/items/d30091274784a28eb872

**Comment by matsumoto616** [comment]
編集していたらあっという間に5GBぐらいのメモリ使用量になりました。。。
![image](https://github.com/increments/qiita-discussions/assets/65050759/058cec1c-63e1-4e97-bff4-319b496ac86a)

**Reply by matsumoto616** [reply]
serviceworker.min.jsというスクリプトが悪さをしていそうです。
![image](https://github.com/increments/qiita-discussions/assets/65050759/83aae031-8f72-435f-8a72-2861b29392f1)

**Comment by tanaka-asami415** [comment]
@matsumoto616 
ご報告ありがとうございます！
#570 と関係する可能性があるかもしれません。併せて調査いたします🙇

**Reply by matsumoto616** [reply]
ありがとうございます。ぜひお願いします！
